canonical-websites/www.ubuntu.com
# New version of the multi-cloud landing page for AB testing

## Done

* Create a new version of the page with form on the left for an A/B test

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure http://www.ubuntu.com-pr-4035.run.demo.haus/engage/multi-cloud-guide and http://www.ubuntu.com-pr-4035.run.demo.haus/engage/multi-cloud-guide2 have forms on opposite sides
